scripts/update-icons-helpers.js

```diff
@@ -13,21 +13,23 @@ function parseSVG(skinIconsFile) {
 
 function getIconKeys(icons = []) {
     const symbolIds = icons.map(symbol => symbol.attributes.id)
+    const skipped = symbolIds.filter(id => !id.startsWith('icon-'));
+
+    if (skipped.length) {
+        console.log(`Skipping ${skipped.length} icons: ${skipped.join(', ')}.`)
```

Comment:
do we need console log?

Reply:
this is just for icons generating script, debug info for devs, so that they notice if something's wrong with the icons...